src/pages/BorrowerProfile/FundedBorrowerProfile.vue

```diff
@@ -326,6 +378,14 @@ export default {
 		if (enabled && exp.version === 'b') {
 			this.enabledExperiment = true;
 		}
+
+		// EXP-MARS-143-Jul2022
+		this.shareCardLanguageVersion = this.$route.query?.utm_campaign;
```

Comment:
this should also be gated on the presence of the string "scle" in the utm_campaign value